### PR TITLE
fix: remove logic to explicitly drop client

### PR DESCRIPTION
**Pull request by nathanielc**
Prior to this change the client needed to be explicitly dropped so that
its message channel would be closed. Now with an update to lspower the
client is closed automatically when the exit notification is received.

### CODE REVIEW

File: Cargo.toml
Comment (Marwes) [inline Cargo.toml:59]: Do we want to wait for this to be released in lspower before merging? be careful not to delete the branch if you want to go ahead.

**Review by Marwes — CHANGES_REQUESTED** [review]

File: Cargo.toml
Comment (nathanielc) [reply Cargo.toml:59]: Yes, I plan to wait for the patch to be release in lspower before merging, this PR is not a critical change to get merge since we have a workaround in place.

**Comment by rockstar** [comment]
This got merged with #427

**Pull request closed without merging**
